Exception in thread "main" java.lang.NullPointerException
	at extest.main(extest.java:369)
